openmc/settings.py

```diff
@@ -125,6 +126,8 @@ class SettingsFile(object):
         Coordinates of the lower-left point of the UFS mesh
     ufs_upper_right : tuple or list
         Coordinates of the upper-right point of the UFS mesh
+    resonance_scattering : ResonanceScattering or iterable thereof
```

Comment:
Ok then I'm cool with allowing either a single element or a collection. As for the docstring, take a look at the spinx-generated documentation. My intuition tells me that the comment will have to be on one line for sphinx to render it properly (as has been the case for a few other docstrings).